src/app/loadout-builder/generated-sets/GeneratedSetButtons.tsx

```diff
@@ -47,8 +49,13 @@ export default function GeneratedSetButtons({
           {t('LoadoutBuilder.CompareLoadout')}
         </button>
       )}
-      <button type="button" className="dim-button" onClick={openLoadout}>
-        {t('LoadoutBuilder.CreateLoadout')}
+      <button
+        type="button"
+        className="dim-button"
+        style={isInLoadout ? { backgroundColor: '#48ba7bbb' } : undefined}
```

Comment:
Can we pull this out into a css class and combine the classnames with clsx?

Reply:
Yep - I was getting errors with adding class names to the `.m.scss` files but that's resolved.